### ISSUE TITLE
Make sure we remove the research vessels from known and likely fishing vessels

### ISSUE THREAD

**Issue by enriquetuya** [GlobalFishingWatch/treniformis#36]
We have found out that there are some research vessels on the `known and likely` fishing vessel list. One example is the `Arc Gorgona` MMSI `730000154`. Maybe the reason is that the AIS message says it is fishing? 

http://www.marinetraffic.com/en/ais/details/ships/shipid:778171/mmsi:730000154/imo:0/vessel:ARC_GORGONA

If this is the case, we need to make sure, that we do not include the research vessels even if their AIS message is fishing.

**Comment by enriquetuya** [comment]
Copying @Bjorn-skytruth comment from slack: 
> I think best solution would be to make a separate Research vessel list with mmsi's but this could require a lot of time looking up individual vessels. By the way how do we want to handle Research vessels that are fishing?

**Comment by Bjorn-skytruth** [comment]
@enriquetuya @pwoods25443 Two research vessels are listed [here](https://docs.google.com/a/skytruth.org/spreadsheets/d/1-YQxRATX1gFm_fGWAJn_-ygmiyujWP7M9V_uA4L5Qzc/edit?usp=sharing). Both vessels identify as fishing on AIS and were likely doing research that involved deploying fishing gear. Do we want to simply remove all research vessels, even those fishing? Or should we somehow try and flag those doing `fishery research` so users will not be concerned by their presence in MPAs for instance.

**Comment by Bjorn-skytruth** [comment]
@enriquetuya The list of research vessels [here](http://www.researchvessels.org/ship_info_results.asp?shipID=n%2Fa&country=n%2Fa&Submit=Begin+Search&affiliation=n%2Fa&grid=n%2Fa&scientists=&speed_cruise=&speed_max=&endurance=&range=&area_wetlab=&area_drylab=&length=) could not be used easily for matching to mmsi. What I need is a list with all identifiers (specifically name, callsign, imo, and flag). I see callsign in the link for each specific vessel but not IMO. So it might be best to try and manually identify the set of research vessels that are making it into GFW, most likely because they are doing fishery research.

**Comment by bitsofbits** [comment]
@Bjorn-skytruth : If it's easy to separate out `fishery_research` and other `research` vessels while doing the matching, we might as well generate two lists and we can decide whether to 
remove them all or just flag the `fishery-research` vessels later. If it's not easy, then I don' t have an opinion one way or another...

**Comment by Bjorn-skytruth** [comment]
@bitsofbits Yes I think it makes sense to identify `fishery-research` vessels specifically though currently the only way I know to approach this is to match vessels individually which is very time consuming.

**Comment by enriquetuya** [comment]
I've sent an email to the European RV infobase contact email to see if we can get a list with MMSIs and last update date. http://www.rvinfobase.eurocean.org/spec/index.jsp

**Comment by davidkroodsma** [comment]
I realize that I should reach out to this group, as they wrote us a while ago about becoming a collaborator. I am assigning this to myself.

**Comment by davidkroodsma** [comment]
@KristinaBoerder @Bjorn-skytruth I scraped the Wikipedia article with a list of research vessels: https://docs.google.com/spreadsheets/d/1DaZQVPSHH0h-3pXJu01vX_8OHGsjsN1KWi3lKeCN8qE/edit#gid=0

someone could go through this and get the mmsi numbers. 

@Bjorn-skytruth  -- what is our goal with getting a research vessel list?

**Comment by Bjorn-skytruth** [comment]
@davidkroodsma I think our main goal is to identify the mmsi numbers of research vessels so that we are able to filter them out of GFW or possibly flag them in the case of a number of fishery research vessels. I believe we do want to include all vessels  which are using fishing gear even for research but we want to identify them so that users will have an explanation for their unusual tracks or activity in MPAs.

**Comment by davidkroodsma** [comment]
@Bjorn-skytruth Great -- thanks.

@KristinaBoerder has put a student on it (10 more points for Kristina!)

**Comment by enriquetuya** [comment]
@davidkroodsma what Bjorn says is correct, the main goal here is to be able to display on the `type` field that the vessels are `research` ones. We have received multiple inqueries from users saying reserach vessels were doing some suspicious activity on MPAs. 

I think the next step after having this list is to have a way of including the `type` on the pipeline so we process that information.

**Comment by davidkroodsma** [comment]
[Research Vessels MMSI_identified.xlsx](https://github.com/GlobalFishingWatch/treniformis/files/628371/Research.Vessels.MMSI_identified.xlsx)

@enriquetuya here is a list pulled together by Kristina's students. We need to send a Global Fishing Watch t-shirt for this. 

The student made this list by Googling every vessel on this list: https://en.wikipedia.org/wiki/List_of_research_vessels_by_country

I have uploaded all vessels that have an mmsi number here to bigquery. You can find it in scratch_david_mmsi_lists.research_vessels

To see how many of these vessels are on our fishing lists:

```
select a.mmsi mmsi, b.vessel_name vessel_name, b.country country, b.notes notes 
from
(SELECT mmsi  FROM 
[world-fishing-827:scratch_david_mmsi_lists.500_cutoff_2012],
[world-fishing-827:scratch_david_mmsi_lists.500_cutoff_2013],
[world-fishing-827:scratch_david_mmsi_lists.500_cutoff_2014],
[world-fishing-827:scratch_david_mmsi_lists.500_cutoff_2015],
[world-fishing-827:scratch_david_mmsi_lists.500_cutoff_2016] group by mmsi) a
inner join
(select mmsi, country, notes, vessel_name, from [scratch_david_mmsi_lists.research_vessels]) b
on a.mmsi = b.mmsi
order by country
```

and the results of this query are here: https://docs.google.com/spreadsheets/d/1_D7MuFG6h4mhHPemj9X7apsJ8A4Rf84RBPCUjwIlcvY/edit#gid=948634962


@enriquetuya do we need anything else? Close issue?

**Comment by enriquetuya** [comment]
@davidkroodsma besides the query mentioned above, have we included a list of research vessels on the repo?

**Comment by davidkroodsma** [comment]
@enriquetuya no -- this is everything we have here. I imagine there are others we could add, but this is all we have right now.

**Comment by davidkroodsma** [comment]
any objection to closing this?
